Wikipedia/UI-V5/WMFArticleFetcher.m

```diff
@@ -20,37 +188,45 @@ @implementation WMFArticleFetcher
 - (instancetype)initWithDataStore:(MWKDataStore*)dataStore {
     self = [super init];
     if (self) {
-        self.dataStore = dataStore;
-        AFHTTPRequestOperationManager* manager = [AFHTTPRequestOperationManager wmf_createDefaultManager];
-        manager.responseSerializer = [AFHTTPResponseSerializer serializer];
-        self.operationManager      = manager;
+        self.operationManager.requestSerializer  = [WMFArticleRequestSerializer serializer];
+        self.operationManager.responseSerializer = [WMFArticleResponseSerializer serializer];
+        self.dataStore                           = dataStore;
     }
     return self;
 }
 
-- (ArticleFetcher*)fetcher {
-    if (!_fetcher) {
-        _fetcher = [[ArticleFetcher alloc] init];
-    }
+- (id)serializedArticleWithTitle:(MWKTitle*)title response:(NSDictionary*)response {
+    MWKArticle* article = [self.dataStore articleWithTitle:title];
```

Comment:
we can leave this as-is for now and make refactoring this a separate task, which IMO we should tackle sooner than later. the mixing of responsibilities and the way articles are written to disk makes this difficult to reason about and test.